app/javascript/v3/helpers/OnboardingHelper.js

```diff
@@ -0,0 +1,16 @@
+export const findMatchingOption = (value, options, defaultValue) => {
+  const match = options.find(
+    option => option.value === value || option === value
+  );
+  return match ? match.value || match : defaultValue;
+};
+
+export const findCompanySizeMatch = (options, size) => {
+  if (!size || Number.isNaN(size)) return undefined;
```

Comment:
Please don't write `return undefined`. A simple return would work here.

Reply:
It throws a lint error. 
<img width="769" alt="image" src="https://github.com/chatwoot/chatwoot/assets/1277421/2f504c42-4dca-4c99-87cf-a4f14e355fba">